ocp4/profiles/coreos-fedramp.profile

```diff
@@ -0,0 +1,17 @@
+documentation_complete: true
```

Comment:
This profile should not be added at this point in time. While Fedramp is the starting target, there are other profiles that are named differently that are essentially the same profile.